.changeset/config.json

```diff
@@ -1,70 +1,13 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.6.4/schema.json",
-  "changelog": [
-    "@changesets/changelog-github",
-    {
-      "repo": "contentful/forma-36"
-    }
-  ],
-  "ignore": [
-    "@contentful/f36-typography",
-    "@contentful/f36-codemod",
-    "@contentful/f36-icon",
-    "@contentful/f36-icons",
-    "@contentful/f36-avatar",
-    "@contentful/f36-image",
-    "@contentful/f36-header",
-    "@contentful/f36-layout",
-    "@contentful/f36-progress-stepper",
-    "@contentful/f36-navlist",
-    "@contentful/f36-navbar",
-    "@contentful/f36-website"
-  ],
+  "changelog": false,
+  "ignore": [],
   "commit": false,
-  "fixed": [
-    [
-      "@contentful/f36-accordion",
-      "@contentful/f36-asset",
-      "@contentful/f36-autocomplete",
-      "@contentful/f36-badge",
-      "@contentful/f36-button",
-      "@contentful/f36-card",
-      "@contentful/f36-collapse",
-      "@contentful/f36-copybutton",
-      "@contentful/f36-core",
-      "@contentful/f36-datetime",
-      "@contentful/f36-datepicker",
-      "@contentful/f36-drag-handle",
-      "@contentful/f36-entity-list",
-      "@contentful/f36-empty-state",
-      "@contentful/f36-forms",
-      "@contentful/f36-icon",
-      "@contentful/f36-header",
-      "@contentful/f36-list",
-      "@contentful/f36-menu",
-      "@contentful/f36-modal",
-      "@contentful/f36-navbar",
-      "@contentful/f36-note",
-      "@contentful/f36-notification",
-      "@contentful/f36-pagination",
-      "@contentful/f36-pill",
-      "@contentful/f36-popover",
-      "@contentful/f36-skeleton",
-      "@contentful/f36-spinner",
-      "@contentful/f36-table",
-      "@contentful/f36-tabs",
-      "@contentful/f36-text-link",
-      "@contentful/f36-tooltip",
-      "@contentful/f36-typography",
-      "@contentful/f36-components",
-      "@contentful/f36-image",
-      "@contentful/f36-avatar"
-    ]
-  ],
   "access": "public",
-  "baseBranch": "main",
+  "baseBranch": "next",
```

Comment:
❓ these changes are only for the time while its a next branch or? So we will not merge it back to the main branch once we have the V5 finalized?

Reply:
This is because sadly we can't have mixed prereleases with releases, so we can only auto publish prereleases from the next branch.
And when merging it back to the main branch from the next, it would mean moving it to stable, for the main branch to auto publish it as stable, that I described on the RELEASES.md.

For alpha packages on the main branch we would still need to manually release sadly, or have two PRs, one pointing to the next branch to have the package published and one to the main branch to add the documentation.